workspace/utilities/com/com-embed-video.xsl

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<!-- COMPONENT: com-embed-video -->
+<xsl:template name="com-embed-video">
+	<xsl:param name="thumbnail" select="thumbnail" />
+	<xsl:param name="is-autoplay" select="true()" />
+	<xsl:param name="autoload" select="'none'" />
+	<xsl:param name="title" select="''" />
+	<xsl:param name="ext-attr" />
+
+	<xsl:variable name="has-thumbnail" select="string-length($thumbnail) != 0" />
+
+
+	<xsl:variable name="attr">
+		<add class="padding-bottom-9_16 relative z-index-1" />
+		<add class="bg-color-black" />
+		<add data-playing-state-follower=".js-auto-oembed-play" />
+		<xsl:copy-of select="$ext-attr" />
+		<add dev-component="com-embed-video" />
+	</xsl:variable>
+
+	<xsl:variable name="attr-video">
+		<add class="absolute fill" />
+		<xsl:call-template name="util-oembed-attr" />
+		<add data-autoload="{$autoload}" />
+		<add data-rel="0" />
+		<xsl:if test="$is-autoplay = true()">
+			<add data-autoplay="1" />
+		</xsl:if>
+		<add class="js-auto-oembed-player" />
+		<add dev-element="video" />
```

Comment:
Missing External attributes

Reply:
Are you talking about those on line 25 ?

Reply:
Added